pkgs/applications/version-management/git-and-tools/gitstatus/default.nix

```diff
@@ -0,0 +1,28 @@
+{callPackage, stdenv, fetchFromGitHub, ...}:
+
+stdenv.mkDerivation rec {
+  name = "gitstatus-${version}";
```

Comment:
use `pname = "gistatus"` instead, mkDerivation will generate the same `name` automatically.

Reply:
oic!

Reply:
fixed